tools/src/main/java/org/neo4j/tools/applytx/DumpRecordsCommand.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.tools.applytx;
+
+import io.airlift.airline.Arguments;
+import io.airlift.airline.Cli;
+import io.airlift.airline.Cli.CliBuilder;
+
+import java.io.PrintStream;
+
+import org.neo4j.helpers.Provider;
+import org.neo4j.kernel.impl.core.Token;
+import org.neo4j.kernel.impl.store.LabelTokenStore;
+import org.neo4j.kernel.impl.store.PropertyKeyTokenStore;
+import org.neo4j.kernel.impl.store.RelationshipTypeTokenStore;
+import org.neo4j.kernel.impl.store.StoreAccess;
+import org.neo4j.kernel.impl.store.record.NodeRecord;
+import org.neo4j.kernel.impl.store.record.PropertyRecord;
+import org.neo4j.kernel.impl.store.record.Record;
+import org.neo4j.kernel.impl.store.record.RelationshipGroupRecord;
+import org.neo4j.kernel.impl.store.record.RelationshipRecord;
+import org.neo4j.tools.console.input.Command;
+import org.neo4j.tools.console.input.ConsoleInput;
+
+import static org.neo4j.tools.console.input.ConsoleUtil.airlineHelp;
+
+/**
+ * Able to dump records and record chains. Works as a {@link ConsoleInput} {@link Command}.
+ */
+public class DumpRecordsCommand implements Command
+{
+    private interface Action
+    {
+        void run( StoreAccess store, PrintStream out );
+    }
+
+    private final Cli<Action> cli;
+    private final Provider<StoreAccess> store;
+
+    @SuppressWarnings( "unchecked" )
+    public DumpRecordsCommand( String commandName, Provider<StoreAccess> store )
+    {
+        this.store = store;
+        CliBuilder<Action> builder = Cli.<Action>builder( commandName )
```

Comment:
Couldn't we supply the constant `"dump"` here, rather than accepting it as a constructor parameter? Would that not have the same effect for `help`? Or am I misunderstanding something around how this works?